Enigmatry.Entry.AspNetCore.Authorization/Requirements/AuthenticatedUserRequirementHandler.cs

```diff
@@ -33,7 +34,7 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
             var requirementName = nameof(requirement);
```

Comment:
nameof(requirement) is just "requirement"
implement .ToString() on the requirement so more data from the requirement can be logged
add userid to the logs.
"user with id does not have correct rights: '...' for access this controller: '...' "